### PR TITLE
Pull request for test

### PR DESCRIPTION
**Pull request by glimberger**
I' d like to add this marvelous code of mine!!

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Test me!!
```
Comment (glimberger) [inline test.txt]: Do you mean this exclamation mark ?

**Comment by CyrilleR** [comment]
Et là?

File: test.txt (hunk 1 above)
Comment (CyrilleR) [inline test.txt]: Yes I dou.

File: dummy_datas.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Lots of data.
```
Comment (glimberger) [inline dummy_datas.txt]: Is there anything to change here ?

Comment (glimberger) [inline dummy_datas.txt]: OK

**Pull request merged**